autogpt_platform/backend/backend/executor/manager.py

```diff
@@ -657,6 +642,52 @@ def on_graph_execution(
         )
         cls.db_client.send_execution_update(result)
 
+        metadata = cls.db_client.get_graph_metadata(
+            graph_exec.graph_id, graph_exec.graph_version
+        )
+        assert metadata is not None
+        outputs = cls.db_client.get_execution_results(graph_exec.graph_exec_id)
+        logger.info(f"{outputs=}")
+
+        # Collect named outputs as a list of dictionaries
+        named_outputs = []
+        for output in outputs:
+            if output.output_data and "name" in output.output_data:
```

Comment:
what's with the output pin named "name" ? 
Is this `AgentOutputBlock` specific can we just filter it using the block_id?

named_outputs = {
    output.output_data
    for output in outputs if output.block_id == AgentOutputBlock().id
}

Reply:
Good idea

Reply:
didn't quite end up doing it this way but did swap the check out